### ISSUE TITLE
Help with Registering Module Events - v1

### ISSUE THREAD

**Issue by marcelogarbin** [nWidart/laravel-modules#408]
Hi, I would like some help with registering new providers.

I'm using Laravel 5.4, so the version of my Laravel modules is v1.0 (1.27.2).

I followed (v2): https://nwidart.com/laravel-modules/v2/advanced-tools/registering-module-events

But it did not work, in that case, how do I use v1, how should I register?

I have the "News" module and "Site" module. I created the event in the "News" module and would like to call it in the "Site" module when I wrote the code `event(new CallEvent($news))`.

Thank you

**Comment by nWidart** [comment]
It's the same process as version 2. This isn't logic related to this package but to laravel, so it's independent.

**Comment by marcelogarbin** [comment]
@nWidart I'm learning to work with events. What I noticed is that when I put the event to listen in the EventServiceProvider App\Core the same worked, but when used in some module does not.

In module a extends the class is called via `use Illuminate\Support\ServiceProvider` https://github.com/nWidart/laravel-modules/blob/b51e5e300ae8bf16019fb7cd1bb13fb5d060f526/src/Commands/stubs/provider.stub#L5

and in the Core: `use Illuminate\Foundation\Support\Providers\EventServiceProvider as ServiceProvider;`
(https://github.com/laravel/laravel/blob/01b9061f69b858a8cf30dccc20f9b10c021ca997/app/Providers/EventServiceProvider.php#L6)

So I made the replacement for the call according to the "Core" and worked as expected.

Sorry for the delay to answer because it was on another project.

Can I use it that way or is it incorrect even if it works?

Thank's
